libs/blocks/quiz-entry/quiz-entry.css

```diff
@@ -178,7 +178,7 @@
 
 .carousel-arrow {
   background-color: #fff;
-  background-image: url('https://main--milo--adobecom.hlx.page/libs/blocks/carousel/img/arrow.svg');
+  background-image: url('https://main--milo--adobecom.aem.page/libs/blocks/carousel/img/arrow.svg');
```

Comment:
Not sure where you want to capture this, but the original is wrong. We should never use Helix URLs on production.

@colloyd do you know where this might be used on production? I want to show that we are making requests to the wrong place.

Reply:
@auniverseaway This is live on this page: https://www.adobe.com/creativecloud.html

Brad handled all of this carousel stuff originally, so he could say more about where the urls for these arrows came from. 

All of that said, there is a redesign of this quiz-entry block going on in which they are pulling the carousel (primarily due to accessibility / ux concerns), so these icons are going to be going away soon.